reference-architectures/deploy-arch-ibm-vsi-fully-configurable.md

```diff
@@ -0,0 +1,170 @@
+---
+
+copyright:
+  years: 2025
+lastupdated: "2025-09-26"
+
+keywords: vsi, virtual server instance, vpc, landing zone, deployable architecture, terraform, fully configurable
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: "Jordan Williams"
+
+# The release that the reference architecture describes
+version: 5.8.1
+
+# Whether the reference architecture is published to Cloud Docs production.
+# When set to false, the file is available only in staging. Default is false.
+production: false
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vsi-ef663980-4c71-4fac-af4f-4a510a9bcf68-global
+
+docs: https://cloud.ibm.com/docs/secure-infrastructure-vpc
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/blob/main/reference-architectures/vsi.svg
+
+related_links:
+  - title: "Cloud automation for Virtual Server Instance on VPC - QuickStart"
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-vsi-ra-qs"
+    description: "The Virtual Server Instance on VPC QuickStart variation is a deployable architecture that provides virtual servers in a VPC in a single region. The QuickStart variation is designed to deploy quickly for demonstration and development."
+  - title: "Cloud automation for Virtual Server Instance on VPC - Fully configurable"
```

Comment:
The DA name and the variations are wrong here!
DA: `Landing zone for applications with virtual servers`
Variation: `Standard - Integrated setup with configurable services`